projects/ccf-rui/src/app/modules/left-sidebar/left-sidebar.component.scss

```diff
@@ -1,7 +1,8 @@
 :host {
   display: flex;
   flex-direction: column;
-  height: 100%;
```

Comment:
Use a `min-height: 100%` to make sure the sidebar takes up all the height (for background purposes mainly). Then set `overflow-y: auto` to get scrolling